modules/defined_classes/manifests/pkg/win_exe_pkg.pp

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+define defined_classes::pkg::win_exe_pkg ($pkg, $creates=$undef, $package=$title, $install_options_string=[]) {
+
+$pkgdir       = lookup('loc_pkg_dir')
```

Comment:
In this case this is a static directory set in hiera. If we pass this as a parameter it would be the same value every time. Alternatively I can set it up as a fact if that is better.

Reply:
If this is a static dir that doesn't change, then hard code it.  Or pass it as a param and do the lookup in the profile.

Reply:
It is a static dir that is called in each of the 3 define type package manifests. My preference is not to hard code the same path multiple times or pass the same value for a parameter multiple times. If for what ever reason this dir doesn't work on a different architecture or OS version, we should have one location to change, so it may stay consistent across all Windows platforms.